docs/Using-Fleet/REST-API.md

```diff
@@ -5076,7 +5076,8 @@ Deletes the session specified by ID. When the user associated with the session n
             "details_link": "https://nvd.nist.gov/vuln/detail/CVE-2009-5155",
             "cvss_score": 7.5,
             "epss_probability": 0.01537,
-            "cisa_known_exploit": false
+            "cisa_known_exploit": false,
+            "cve_published": "2022-01-01 12:32:00",
```

Comment:
Nit, invalid JSON :)

Reply:
Done in https://github.com/fleetdm/fleet/pull/10434/commits/2aedfc2750e65dfe7ea98e98184122986b6d339d